crescent/ext/tasks/loop.py

```diff
@@ -11,7 +11,7 @@
 
 class Loop(Task):
     def __init__(self, callback: TaskCallbackT, delay_seconds: float) -> None:
-        self.delay_seconds = delay_seconds
+        self.delay_seconds: float = delay_seconds
```

Comment:
Is this necessary?

Reply:
Yeah :(
Pyright --veryify-types doesn't allow any types to be inferred in constructors when the variable is also used somewhere else in the class

Reply:
Pyright L